app/controllers/test_controller.rb

```diff
@@ -82,20 +82,6 @@ def new_link
     new_media 'link'
   end
 
-  def new_source
```

Comment:
we should keep the option of creating a source from the test controller... So maybe this test needs to make sure that the source is creating correctly, and remove the `ProjectSource` part?

Reply:
I removed it as it designed to create a `ProjectSource` not Source https://github.com/meedan/check-api/blob/develop/app/controllers/test_controller.rb#L85-L97 
Shall I change the logic to create `Source` instead of `ProjectSource`?

Reply:
yes, I think that's the best approach :pray: